### PR TITLE
Track simple payments separately and normal payments when select payment screen is reused

### LINKED ISSUE #6695: [Mobile Payments] Figure out how to track Simple payments separately and normal payments when SelectPayment screen is reused
We rename current Simple payments events to PAYMENTS_ and add property inside to reflect the flow:

* PAYMENTS_FLOW_COLLECT
* PAYMENTS_FLOW_COMPLETED
* PAYMENTS_FLOW_FAILED

And add params:

* flow -> simple_payment 
* flow -> order_payment

pdfdoF-TT-p2

- [x]  Add a note to the tracks that  3 events were renamed pdfdoF-TT-p2#comment-1818

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #6695

### Description

* Moves "refund" package to the "payments" package
* Renames:
    * `SIMPLE_PAYMENTS_FLOW_COMPLETED` -> `PAYMENTS_FLOW_COMPLETED`
    * `SIMPLE_PAYMENTS_FLOW_COLLECT` -> `PAYMENTS_FLOW_COLLECT`
    *  `SIMPLE_PAYMENTS_FLOW_FAILED` -> `PAYMENTS_FLOW_FAILED`
* To the aforementioned events params are added:
   * `flow: simple_payment` or `flow: order_payment`

### Testing instructions

* Run both simple payments flow and order payment flow. Click on "cash", "card" or "share payment link" on the payment selection scren
* Use logcat filtered by `Tracked`
* Notice all renamed events and newly added params

### CODE REVIEW

**Review by AnirudhBhat — APPROVED** [review]
Thanks for working on this @kidinov! I've tested the changes and it worked as expected. The code changes look good as well. I've left one minor comment and it's not blocking at all. Feel free to merge the changes regardless of what you think about my comment. :shipit:

File: WooCommerce/src/main/kotlin/com/woocommerce/android/analytics/AnalyticsEvent.kt [hunk 1]
```diff
@@ -141,11 +141,13 @@ enum class AnalyticsEvent(val siteless: Boolean = false) {
     FILTER_ORDERS_BY_STATUS_DIALOG_APPLY_FILTER_BUTTON_TAPPED,
     FILTER_ORDERS_BY_STATUS_DIALOG_OPTION_SELECTED,
 
+    // -- Payments
+    PAYMENTS_FLOW_COMPLETED,
+    PAYMENTS_FLOW_COLLECT,
+    PAYMENTS_FLOW_FAILED,
+
     // -- Simple Payments
     SIMPLE_PAYMENTS_FLOW_STARTED,
```
Comment (AnirudhBhat) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/analytics/AnalyticsEvent.kt:150]: ❓ What are your thoughts on removing the `SIMPLE_PAYMENTS_FLOW_STARTED` and  `SIMPLE_PAYMENTS_FLOW_CANCELED` events and replacing them with a generic `PAYMENTS_FLOW_STARTED` and  `PAYMENTS_FLOW_CANCELED` event with the property `simple_payment` or `order_payment`?

Definitely not needed in this PR and if you think it's worth doing these changes. It can be taken in a separate PR.

Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/analytics/AnalyticsEvent.kt:150]: Good point! I'll p2 about that to align with ios and so people can adjust they funnels
